openshift/deploy/service/shipit.yml

```diff
@@ -14,7 +14,7 @@ objects:
       name: ${PREFIX}shipit
     spec:
       ports:
-        - name: ${PREFIX}shipit
+        - name: ${PREFIX}shipit-web
```

Comment:
shouldn't the selector be updated too/instead?

Reply:
yessir